misc/solr/Dockerfile

```diff
@@ -1,28 +1,38 @@
 # syntax=docker/dockerfile:1
 # check=skip=InvalidBaseImagePlatform
 
+### Git downloader for search_api_solr
+FROM alpine AS git-downloader
+
+RUN apk --update --no-cache add git
+
+WORKDIR /tmp
+
+RUN git clone https://git.drupalcode.org/project/search_api_solr.git
+
+
 #
 # AMD64
 #
-FROM --platform=linux/amd64 solr:8.11-slim AS solr-8-amd64
+FROM --platform=linux/amd64 solr:9.8.1-slim AS solr-9-amd64
 
 ENV ARCH=amd64
 
 #
 # ARM64
 #
-FROM --platform=linux/arm64 arm64v8/solr:8.11-slim AS solr-8-arm64
+FROM --platform=linux/arm64 arm64v8/solr:9.8.1-slim AS solr-9-arm64
 
 ENV ARCH=arm64
 
 #
 # Final
 #
-FROM solr-8-${TARGETARCH} AS solr
+FROM solr-9-${TARGETARCH} AS solr
```

Comment:
as 9.8.1-slim is already available for both architectures, you could use it as FROM value and remove the amd64 and arm64 ^^